op-bindings/predeploys/addresses.go

```diff
@@ -56,6 +58,7 @@ func IsProxied(predeployAddr common.Address) bool {
 }
 
 func init() {
+	Predeploys["LegacyERC20ETHAddr"] = &LegacyERC20ETHAddr
```

Comment:
Sorry if my comment was unclear, it can't go in the mapping below, I think it makes sense to put it in a different file, could be in this package tho

Reply:
no worries, i just realized when ci ran (also til the eth address is not same post-bedrock). Adjusting it rn